src/main/java/org/jabref/logic/externalfiles/ExternalFilesContentImporter.java

```diff
@@ -17,9 +19,9 @@ public ExternalFilesContentImporter(ImportFormatPreferences importFormatPreferen
         this.importFormatPreferences = importFormatPreferences;
     }
 
-    public ParserResult importPDFContent(Path file) {
+    public ParserResult importPDFContent(Path file, BibDatabaseContext context, FilePreferences preferences) {
```

Comment:
Name the `preferences` variable `filePreferences` - also in the method `importDatabase`